src/app/pages/Bookmarks/Bookmarks.module.css

```diff
@@ -0,0 +1,26 @@
+::-webkit-scrollbar {
+  display: none;
+}
+
+.container {
+  background-color: var(--bg-primary);
+  display: grid;
+  height: 100vh;
+  overflow: auto;
+}
+
+.header {
+  padding-top: 2.5rem;
+  padding-left: 1rem;
+  position: sticky;
+  top: 0;
+  z-index: 100;
+  background-color: var(--bg-primary);
+}
+
+.cardWrapper {
+  display: grid;
+  padding: 1em;
+  gap: 1em;
+  overflow: scroll;
+}
```

Comment:
Maybe some responsiveness, with a `repeat(auto-fit, minmax())` type grid?
https://github.com/marwinburesch/moviescout/pull/26#pullrequestreview-741660803

Reply:
Resolved in commit [685c03a](https://github.com/marwinburesch/moviescout/pull/27/commits/685c03a89f1c642ec291f67248383e8374ac2fc2)